controllers/accountclaim/accountclaim_controller_test.go

```diff
@@ -313,3 +314,347 @@ func (p *possiblyErroringFakeCtrlRuntimeClient) Update(
 	}
 	return p.Client.Update(ctx, acc)
 }
+
+var _ = Describe("Mutiple AccountPools Claim", func() {
+	var (
+		namespace = awsv1alpha1.AccountCrNamespace // Can't set custom namespace as getUnclaimedAccount has this hardcoded
+
+		defaultAccountName     = "default-account"
+		defaultClaimName       = "default-accountclaim"
+		defaultAccountPoolName = "my-default-accountpool"
+
+		sqAccountName     = "service-quota-account"
+		sqClaimName       = "service-quota-accountclaim"
+		sqAccountPoolName = "my-service-quota-accountpool"
+
+		accounts      []*awsv1alpha1.Account
+		accountClaims []*awsv1alpha1.AccountClaim
+		configMap     *v1.ConfigMap
+
+		r              *AccountClaimReconciler
+		ctrl           *gomock.Controller
+		req            reconcile.Request
+		reconcileCount = 2
+	)
+
+	err := apis.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Printf("failed adding apis to scheme in account controller tests")
+	}
+	localmetrics.Collector = localmetrics.NewMetricsCollector(nil)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		myYaml := `
```

Comment:
Does this work? It seems to be missing any of the 'multiline' markers for yaml like `|`.